chainHead: Allow single connection calls and enforce order of events (#137)
paritytech/json-rpc-interface-spec: src/api/chainHead_unstable_call.md

The progress of the operation is indicated through `operationCallDone`, `operationInaccessible`, or `operationError` notifications generated on the corresponding `chainHead_unstable_follow` subscription.

> This function must be called from the same connection as the corresponding `chainHead_unstable_follow` subscription. The `operationId` is guaranteed to be returned before any operation notifications on the corresponding `chainHead_unstable_follow` subscription.

Reviewer: ```suggestion
This function must be called from the same connection as the corresponding `chainHead_unstable_follow` subscription. The `operationId` is guaranteed to be returned before any operation notification on the corresponding `chainHead_unstable_follow` subscription.
```

> This function can only be called from the same connection as the one that made the `chainHead_unstable_follow` subscription. The `operationId` is guaranteed to be generated before the `operationCallDone`, `operationInaccessible`, or `operationError` notifications are generated on the corresponding `chainHead_unstable_follow` subscription.

Reviewer 2: ```suggestion
This function must be called from the same connection as the corresponding `chainHead_unstable_follow` subscription. The `operationId` is guaranteed to be returned before any operation notifications on the corresponding `chainHead_unstable_follow` subscription.
```

Just trying to condense it down a bit :)
